docs/_posts/examples/3400-01-02-cluster.html

```diff
@@ -31,56 +30,55 @@
         clusterRadius: 50 // Radius of each cluster when clustering points (defaults to 50)
     });
 
-    // Use the earthquakes source to create five layers:
-    // One for unclustered points, three for each cluster category,
-    // and one for cluster labels.
     map.addLayer({
-        "id": "unclustered-points",
-        "type": "symbol",
-        "source": "earthquakes",
-        "filter": ["!has", "point_count"],
-        "layout": {
-            "icon-image": "marker-15"
+        id: "clusters",
+        type: "circle",
+        source: "earthquakes",
+        filter: ["has", "point_count"],
+        colorSpace: "lab",
```

Comment:
I don't think you need `colorSpace` unless you're using an exponential function (colorSpace is used for interpolation, and `categorical` functions don't interpolate color)